Makefile

```diff
@@ -79,8 +79,9 @@ test: clean-last-testrun test-unit test-integration test-cli
 test-unit:
 	$(GO) test ./...
 
-test-integration: build
-	$(GO) test -timeout 20m -tags=integration ./...
+test-integration: 
+	$(GO) build -o $(PORTER_HOME)/testplugin ./cmd/testplugin
+	PROJECT_ROOT=$(shell pwd) $(GO) test -timeout 20m -tags=integration ./...
```

Comment:
I was not able to copy testplugin binary from `bin` directory, since relative path not working. Even if we run `make test-integration` from project root directory, `os.Getwd` will be parent directory of test file. So, I needed `PROJECT_ROOT` to access `bin` directory.